services/docker-compose-build.yml

```diff
@@ -87,6 +87,22 @@ services:
         org.label-schema.vcs-url: "${VCS_URL}"
         org.label-schema.vcs-ref: "${VCS_REF}"
 
+  static-webserver:
+    image: local/static-webserver:${BUILD_TARGET:?build_target_required}
+    build:
+      context: ../
+      dockerfile: services/static-webserver/Dockerfile
+      cache_from:
+        - local/static-webserver:${BUILD_TARGET:?build_target_required}
+        - ${DOCKER_REGISTRY:-itisfoundation}/static-webserver:cache
```

Comment:
Do we need the cache for this new service? This is for multi-stage Dockefile with a stage called ``cache``, which based on ``services/static-webserver/Dockerfile``, does NOT have.

We actually should deprecate them all. It is an old trick taht is not needed anymore

Reply:
removed caching from here